src/org/opendatakit/briefcase/reused/transfer/CentralServer.java

```diff
@@ -65,6 +65,12 @@ private static String buildSessionPayload(Credentials credentials) {
     );
   }
 
+  public static String cleanUrl(String url) {
+    return url.contains("/#/")
+        ? url.substring(0, url.indexOf("/#/"))
+        : url;
```

Comment:
May I suggest this?
```
    int i = url.indexOf("/#/");
    return i == -1 
        ? url 
        : url.substring(0, i);
```
It only searches the string once, and more important, doesn’t duplicate the search argument.